cmd/gossamer/config.go

```diff
@@ -465,6 +445,47 @@ func setDotGlobalConfigFromToml(tomlCfg *ctoml.Config, cfg *dot.GlobalConfig) {
 	}
 }
 
+func setDotBasepathConfig(ctx *cli.Context, tomlConfig *ctoml.Config, cfg *dot.GlobalConfig) {
+	setDotBasepathConfigFromToml(tomlConfig, cfg)
+	setDotBasepathConfigFromFlags(ctx, cfg)
+}
+
+func setDotBasepathConfigFromToml(tomlCfg *ctoml.Config, cfg *dot.GlobalConfig) {
+	if tomlCfg == nil {
+		return
+	}
+
```

Comment:
nit do we need this check?

Reply:
Based on the usage of the function, we do not need to use it at this time. However, this function was written with the goal of emulating the call to `setDotGlobalConfig`, which does have a check for a `nil` TOML config. Ref: https://github.com/ChainSafe/gossamer/blob/v0.6.0/cmd/gossamer/config.go#L421